apps/editing-toolkit/editing-toolkit-plugin/wpcom-global-styles/modal.js

```diff
@@ -1,15 +1,28 @@
 /* global wpcomGlobalStyles */
 
 import { Button, Modal } from '@wordpress/components';
-import { useState } from '@wordpress/element';
+import { useDispatch, useSelect } from '@wordpress/data';
+import { useEffect } from '@wordpress/element';
 import { __ } from '@wordpress/i18n';
 import React from 'react';
 import image from './image.svg';
 
 import './modal.scss';
 
 const GlobalStylesModal = () => {
-	const [ isVisible, setIsVisible ] = useState( true );
+	const isVisible = useSelect(
+		( select ) => select( 'automattic/wpcom-global-styles' ).isModalVisible(),
```

Comment:
Nitpick, you can remove the parenthesis of the arrow function since there is only one argument.

Reply:
It's enforced by our JS coding standards:

<img width="737" alt="Screen Shot 2022-10-04 at 09 59 49" src="https://user-images.githubusercontent.com/1233880/193766296-6cd4e403-1b4e-4ef7-af0f-5fbfc2653f1d.png">